lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -236,27 +237,28 @@ sub info {
 
 sub send_command {
     my ($self, %args) = @_;
-    return if (!defined $args{command});
+    return undef if (!defined $args{command});
 
     if (!grep { $args{command} eq $_ } COMMANDS) {
         my $msg = 'Trying to issue unknown command "%s" for worker "%s:%n"';
         log_error(sprintf($msg, $args{command}, $self->host, $self->instance));
-        return;
+        return undef;
```

Comment:
Is there a functional difference? Just wondering... it definitely looks clearer.

Reply:
Explicit return undef avoids context sensitivity